preupg/creator/ui_helper.py

```diff
@@ -253,19 +260,37 @@ def _create_group_ini(self):
         config.set(section, 'group_title', 'Title for %s ' % self.get_group_name())
 
         file_name = 'group.ini'
-        group_ini = os.path.join(self.get_upgrade_path(), self.get_group_name(), file_name)
+        group_ini = os.path.join(self.get_upgrade_path(),
+                                 self.get_group_name(),
+                                 file_name)
         if os.path.exists(group_ini):
             return
-        try:
-            f = open(group_ini, 'wb')
-            config.write(f)
-            f.close()
-        except IOError:
-            print ('We have a problem with writing %s file to disc' % group_ini)
-            raise
+        UIHelper._write_to_file_content(group_ini, config)
+
+    def _create_properties_ini(self):
+        """
+        Create properties.ini inside module set directory in format:
+
+        [preupgrade-assistant-modules]
+        srcMajorVersion =
+        dstMajorVersion =
+        """
+        file_name = 'properties.ini'
+        section = 'preupgrade-assistant-modules'
+
+        config = ConfigParser.RawConfigParser()
+        config.add_section(section)
+        config.set(section, 'srcMajorVersion', '')
+        config.set(section, 'dstMajorVersion', '')
```

Comment:
Values should be filled by user during creation of modules in same way as other input is filled.
@AloisMahdal: what do you think about that Lojzo? I am keeping decision on you, but from my side it should be done during creation of modules.